### PR TITLE
[WIP] Add Jupyter Server Extension for Play

### PR DESCRIPTION
**Pull request by yuvipanda**
Lots of work left to do!

### CODE REVIEW

File: applications/play-extension/MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,5 @@
+include README.md
+include setup.py
+recursive-include nteract_on_jupyter *
```
Comment (rgbkrk) [inline applications/play-extension/MANIFEST.in:3]: need to swap this over to `play-extension`

File: applications/play-extension/Pipfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+jupyter = "*"
+
+
+[dev-packages]
+
+
+
+[requires]
+
+python_version = "3.6"
```
Comment (rgbkrk) [inline applications/play-extension/Pipfile:19]: Oooooh, I've not seen a `Pipfile` yet.

Comment (rgbkrk) [reply applications/play-extension/Pipfile:19]: Part of [pipenv goodness](https://github.com/pypa/pipfile)!

File: applications/play-extension/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# nteract on jupyter
+
+nteract as an extension to the jupyter notebook server
+
+## Installation
+
+```
+pip install nteract_on_jupyter
```
Comment (rgbkrk) [inline applications/play-extension/README.md]: Ok I'll stop commenting on copy pasta that needs updates here. 😉

File: applications/play-extension/play_on_jupyter/package.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "nteract-on-jupyter",
```
Comment (rgbkrk) [inline applications/play-extension/play_on_jupyter/package.json]: In order for the app to bootstrap properly, you'll want to change this one the most. 😉

**Comment by rgbkrk** [comment]
Working my way towards fully working. Next to fix:

```
[W 15:04:52.365 NotebookApp] Error loading server extension play_on_jupyter
    Traceback (most recent call last):
      File "/usr/local/lib/python3.6/site-packages/notebook/notebookapp.py", line 1454, in init_server_extensions
        func(self)
      File "/Users/kylek/code/src/github.com/nteract/nteract/applications/play-extension/play_on_jupyter/extension.py", line 69, in load_jupyter_server_extension
        (ujoin(base_url, config.page_url, r"/static/(.*)"), FileFindHandler, {
    NameError: name 'FileFindHandler' is not defined
```

**Comment by rgbkrk** [comment]
Since we're on track with the editor PR in nteract-on-jupyter as the way forward, we can probably close this right?

**Comment by yuvipanda** [comment]
Yep, we can close this!

**Pull request closed without merging**